From 1041b2d397a0663a3c77a5382ed99d91d4022e01 Mon Sep 17 00:00:00 2001
From: Johan Hovold <johan+linaro@kernel.org>
Date: Tue, 3 Jan 2023 10:09:10 +0100
Subject: [PATCH 043/230] arm64: dts: qcom: sm8250: clean up wcd938x codec node

Clean up the wcd938x codec node somewhat by adding newline separators,
reordering properties and renaming it 'audio-codec'.

Signed-off-by: Johan Hovold <johan+linaro@kernel.org>
---
 arch/arm64/boot/dts/qcom/sm8250-mtp.dts | 16 ++++++++++------
 1 file changed, 10 insertions(+), 6 deletions(-)

diff --git a/arch/arm64/boot/dts/qcom/sm8250-mtp.dts b/arch/arm64/boot/dts/qcom/sm8250-mtp.dts
index b741b7da1af..0991b34a8e4 100644
--- a/arch/arm64/boot/dts/qcom/sm8250-mtp.dts
+++ b/arch/arm64/boot/dts/qcom/sm8250-mtp.dts
@@ -23,18 +23,16 @@ aliases {
 		serial0 = &uart12;
 	};
 
-	chosen {
-		stdout-path = "serial0:115200n8";
-	};
-
-	wcd938x: codec {
+	wcd938x: audio-codec {
 		compatible = "qcom,wcd9380-codec";
-		#sound-dai-cells = <1>;
+
 		reset-gpios = <&tlmm 32 GPIO_ACTIVE_LOW>;
+
 		vdd-buck-supply = <&vreg_s4a_1p8>;
 		vdd-rxtx-supply = <&vreg_s4a_1p8>;
 		vdd-io-supply = <&vreg_s4a_1p8>;
 		vdd-mic-bias-supply = <&vreg_bob>;
+
 		qcom,micbias1-microvolt = <1800000>;
 		qcom,micbias2-microvolt = <1800000>;
 		qcom,micbias3-microvolt = <1800000>;
@@ -44,6 +42,12 @@ wcd938x: codec {
 		qcom,mbhc-headphone-vthreshold-microvolt = <50000>;
 		qcom,rx-device = <&wcd_rx>;
 		qcom,tx-device = <&wcd_tx>;
+
+		#sound-dai-cells = <1>;
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
 	};
 
 	thermal-zones {
-- 
2.39.0

